packages/nimble-components/src/rich-text-editor-viewer/specs/README.md

```diff
@@ -0,0 +1,377 @@
+
+**_Note_**: This prototype is not made in a responsive fashion for smaller screens and is intended only for testing the specific functionalities listed
+above.
+
+### States
+
+_Button State_
+
+The buttons in the toolbar will indicate the formatting status of the selected text in the editor. For example, if the text in the editor
+is bold and the cursor is placed on the text, the bold button will be indicated as a active state. The active state is indicated as `block`
+appearance of the [`nimble-button`](https://nimble.ni.dev/storybook/?path=/docs/button--docs) and inactive state is in `ghost`
+appearance. Refer the below sample screenshot with active bold state.
+
+![Bold Button State](./spec-images/button-state.png)
+
+### Accessibility
+
+[Tiptap Accessibility](https://tiptap.dev/guide/accessibility)
+
+_Focus_
+
+-   Focus state of the editor will be the same as the `nimble-text-area`.
+-   Focus state of the buttons will be the same as the `nimble-button`.
+
+_Keyboard accessibility and shortcuts for text formatting_
+
+All the basic keyboard accessibility for the tiptap rich text editor is mentioned in their
+[keyboard shortcut](https://tiptap.dev/api/keyboard-shortcuts#predefined-keyboard-shortcuts) page. To facilitate easy access, listed down
+the supported text formatting keyboard shortcuts in the tiptap editor.
+
+| Key                     | Behavior                                                 |
+| ----------------------- | -------------------------------------------------------- |
+| Tab                     | To reach the editor                                      |
+| Shift + Arrow keys      | To select the group of texts in the editor               |
+| Ctrl + B                | To make the selected text bold                           |
+| Ctrl + I                | To make the selected text italic                         |
+| Ctrl + Shift + 7        | To enable the focused paragraph a numbered list          |
+| Ctrl + Shift + 8        | To enable the focused paragraph a bulleted list          |
+| Tab (For lists)         | To create a sub point within a bulleted or numbered list |
+| Shift + Tab (For lists) | To remove a sub point from a bulleted or numbered lists  |
+
+_Keyboard navigation with toolbar buttons focused_
+
+| Key          | Behavior                                                                       |
+| ------------ | ------------------------------------------------------------------------------ |
+| Space, Enter | Enable the selected text formatting feature and return the focus to the editor |
+| Shift + Tab  | To reach the editor back to focus                                              |
+
+_Note_: All other keyboard interaction determined by the slotted element will not be defined in this document.
+
+### Globalization
+
+Currently, this component does not have any string data to be localized. In the footer menu for text formatting, only icons will be displayed. However,
```

Comment:
The `nimble-tooltip` is an experimental component, so I wouldn't recommend using it here.

Also, I think it should be a requirement to have tooltips (we've been using the `title` attribute to accomplish this) and accessible text for each icon button. There is a spec in progress for how to configure localizable strings on a component, so once the infrastructure work is complete, I'd expect this component to plug into the shared mechanism for providing localized strings. See #1272

Reply:
Thanks for the information on localization. I have updated the same in the spec.